tests/Command/RunCommandTest.php

```diff
@@ -509,9 +519,13 @@ public function testExecuteUnEncryptedJobData(): void
             'command' => $command->getName(),
         ]);
 
-        self::assertTrue($testHandler->hasErrorThatContains(
-            'Job "' . $job->getId() . '" ended with encryption error: "Internal Server Error occurred."'
-        ));
+        var_dump($testHandler->getRecords());
+
+        self::assertTrue($testHandler->hasErrorThatContains(sprintf(
+            'Job "%s" ended with encryption error: '.
+            '"Invalid cipher text for key #foo Value "bar" is not an encrypted value."',
+            $job->getId(),
+        )));
```

Comment:
Hazi to tady tenhle error i kdyz ten token ted encryptujes? A je tam zapomenuty var_dump.

Reply:
sry, var_dump jsem zapomenel.. pouzil jsme ho prave na dohledani toho erroru (predtim to jako error psalo "internal exception")